spec/views/posts_show_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
```

Comment:
- [ ] There are some tests that fail when I run `rspec spec/views/`. Please ensure all the tests are passed successfully.

<img width="640" alt="image" src="https://user-images.githubusercontent.com/77190203/180201428-093f2bfd-f85a-44cd-a72e-d4079ea149c8.png">